.gitlab-ci.yml

```diff
@@ -1255,6 +1267,25 @@ latest_release:
     - inv -e docker.publish --signed-pull --signed-push datadog/agent:${RELEASE}-jmx datadog/agent:latest-jmx
     - inv -e docker.publish --signed-pull --signed-push datadog/dogstatsd:${RELEASE} datadog/dogstatsd:latest
 
+#
+# Use this step to delete a tag of a given image
+# - remove the leading dot from the name
+# - set the IMAGE and TAG envvars
+# - in the gitlab pipeline view, trigger the step (in the first column)
+#
+.delete_docker_tag:
+  <<: *docker_delete_tag_job_definition
```

Comment:
Is there a reason the job definition should be separated like this? I don't see it being used elsewhere, so might make readability easier to just include it here? 🙂

Reply:
yes I think in this case there is no good reason to separate the job definition, thanks for pointing out 🙂